src/coupling/balance.hh

```diff
@@ -260,42 +262,34 @@ public:
 			start_source_assembly(idx);
 	}
 
+	
 	/**
-	 * Adds elements into matrix for computing mass.
+	 * Adds elements into matrix for computing (outgoing) flux.
```

Comment:
for computing mass.
... document shape of matrix and vector for every kind of the add_* routine.